content/source/docs/enterprise/api/_template.md

```diff
@@ -93,8 +93,6 @@ This GET endpoint can optionally return related resources, if requested with [th
 Resource Name      | Description
 -------------------|------------
 `organization`     | The full organization record.
-`latest_run`       | Additional information about the last run.
-`latest_run.plan ` | The plan used in the last run.
```

Comment:
Could you please replace these with `current_run` equivalents, so the template will still have multiple resource type to show how to do it?

Reply:
Good call.